css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,18 @@
+.hover {
+  padding: 15px 35px;
+  font-size: 24px;
+}
+
+.hover:hover {
+  color: white;
+  background-color: black;
+}
```

Comment:
The cursor should change to a pointer when hovering over the button. Also, the button's spacing should be consistent with the example.

---

![image](https://user-images.githubusercontent.com/16143771/158656026-e125e4e9-903d-47ce-8552-9784979ae100.png)

---

![image](https://user-images.githubusercontent.com/16143771/158656093-6596ed7f-576e-4646-8dd7-08f12dcf4ea5.png)

Reply:
![css-pseudo-classes-2](https://user-images.githubusercontent.com/43438792/158657745-d79a4006-ddcf-41e2-bf94-b92456be8bb1.gif)

Reply:
@K-Bowler What about the spacing?